Sofa/Component/LinearSolver/Iterative/src/sofa/component/linearsolver/iterative/MatrixLinearSolver.h

```diff
@@ -54,7 +54,7 @@ template<class Matrix, class Vector>
 class BaseMatrixLinearSolver : public sofa::core::behavior::LinearSolver
 {
 public:
-    SOFA_CLASS(SOFA_TEMPLATE2(BaseMatrixLinearSolver,Matrix,Vector), sofa::core::behavior::LinearSolver);
+    SOFA_ABSTRACT_CLASS(SOFA_TEMPLATE2(BaseMatrixLinearSolver,Matrix,Vector), sofa::core::behavior::LinearSolver);
```

Comment:
Is that mandatory for the compilation to succeed?

Reply:
ah no, it was during my investigation... but it seems anyway more correct as SOFA_CLASS adds the New() feature. And you are not supposed to New() a BaseMatrixLinearSolver IMO

Reply:
It's what I thought. Thanks